src/components/Layout/Header/Header.component.jsx

```diff
@@ -15,6 +16,8 @@ const Header = () => {
   const [click, setClick] = useState(false);
   const handleClick = () => setClick(!click);
   const closeMenu = () => setClick(false);
+  //const [showSearch, setShowSearch] = useState(false);
+  //const handleSearch = () => setShowSearch(!showSearch);
```

Comment:
Unless you plan to add these lines back in, remove them so that they don't add clutter.